torch/csrc/jit/script/compiler.cpp

```diff
@@ -2057,19 +2059,37 @@ struct to_ir {
       }
       return std::make_shared<SimpleValue>(expr);
     } else if(auto getattr = dynamic_cast<GetAttrValue*>(sv.get())) {
-      if (apply.attributes().size() > 0) {
-        throw ErrorReport(loc) << "getattr takes no keyword arguments";
-      }
-      if (apply.inputs().size() != 2) {
-        throw ErrorReport(loc) << "getattr expects 2 inputs";
-      }
+      checkApplyExpr(apply, loc);
       auto obj = emitSugaredExpr(apply.inputs()[0], 1);
       auto selector = apply.inputs()[1];
       if (selector.kind() != TK_STRINGLITERAL) {
         throw ErrorReport(loc) << "getattr's second argument must be a string literal";
       }
       const std::string& name = StringLiteral(selector).text();
       return obj->attr(apply.range(), method, name);
+    } else if (auto isinst = dynamic_cast<IsInstValue*>(sv.get())) {
```

Comment:
Do we want `isinstance` to be a thing for both the Python and string frontend? If we just want it in Python then adding it as a `SugaredValue` to `script/init.cpp` is probably a better path.